helm/nessie/templates/configmap.yaml

```diff
@@ -111,14 +111,22 @@ data:
     {{- else if eq .Values.tracing.sample "none" -}}
     {{- $_ = set $map "quarkus.otel.traces.sampler" "always_off" -}}
     {{- else -}}
-    {{- $_ = set $map "quarkus.otel.traces.sampler=" "parentbased_traceidratio" -}}
+    {{- $_ = set $map "quarkus.otel.traces.sampler" "parentbased_traceidratio" -}}
```

Comment:
Fixing this bug.

Reply:
And maybe the new code-style issues ;) ;)